cardano-rt-view/src/Cardano/Benchmarking/RTView/GUI/Updater.hs

```diff
@@ -152,6 +155,27 @@ updatePeersList peersInfo peersList = do
       ]
   element peersList # set children peersItems
 
+updateErrorsList
+  :: [NodeError]
+  -> Element
+  -> UI Element
+updateErrorsList nodeErrors errorsList = do
+  errors <- forM nodeErrors $ \(NodeError timeStamp sev msg) -> do
+    let className :: String
+        className = case sev of
+                      Warning   -> "warning-message"
+                      Error     -> "error-message"
+                      Critical  -> "critical-message"
+                      Alert     -> "alert-message"
+                      Emergency -> "emergency-message"
+                      _         -> ""
```

Comment:
why not assign a default class for all other severities?

Reply:
Because technically we don't need it: this list contains only errors (we already filtered it earlier). So I keep this empty class to avoid incomplete pattern matching only.